Daniel-L-Ross/parkbook
# Dr park list

# Description

This branch includes a new provider to get parks data, and corresponding components to render all parks in a list. 

Fixes #13 

## Type of change


# Testing Instructions

Run a json server for the api
npm start for the app
All parks should render in a list on page load. 


# Checklist:
